App.tsx

```
@@ -43,6 +43,7 @@ function App(): React.JSX.Element {
 
   return (
       <View style={styles.container}>
+        <Tex><Text><Text></Text></Text></Tex>
```

ESLint

[error] 'Tex' is not defined.

(react/jsx-no-undef)